### ISSUE TITLE
Inclusion of locale directory installing latest version from Github

### ISSUE THREAD

**Issue by mondeja** [celery/django-celery-beat#324]
### Summary:
It could be nice that I could run `pip install git+https://github.com/celery/django-celery-beat.git#egg=django_celery_beat` and the `locale` directory were be included inside `django_celery_beat` directory of my virtualenv.

* Celery Version: 4.4.2
* Celery-Beat Version: 2.0.0

### Exact steps to reproduce the issue:
1. Install the library with `pip install django-celery-beat` inside virtual environment.
2. See that locale directory is not found.
3. Install the library with `pip install git+https://github.com/celery/django-celery-beat.git#egg=django_celery_beat`.
4. See again that locale directory is not found.

### Detailed information
For now, I'm installing `django-celery-beat` from Pypi. This not include locales, and this is the expected behaviour. For locale directory inclusion, I'm currently executing a script that download the selected locales from the repository and including them in the directory of the library inside my virtualenv.

Other packages like `sphinx_rtd_theme` [includes the locales in `MAINFEST.in`](https://github.com/readthedocs/sphinx_rtd_theme/blob/master/MANIFEST.in) using `recursive-include`, but doesn't include it on source code tarball distributed through Pypi. See [this](https://github.com/readthedocs/sphinx_rtd_theme/issues/888) for a test case.

It would be nice that we can include the locales, at least, installing `django-celery-beat` through Github. It doesn't make sense the inclusion of locales inside source code tarball distributed through Pypi because are not stable, especially the chinese translation. For this, we need only a couple of changes:

- Include `locale` directory inside `django_celery_beat` directory.
- Include the line `recursive-include django_celery_beat/locale *.po *.mo` inside MANIFEST.in
- In release process, exclude the locales directory.

I can accomplish this with a pull, but I need help with release process updating, because there are not any public script included in the repository.

**Comment by belongwqz** [comment]
waiting

**Comment by mondeja** [comment]
I've resolved this problem with:

1. Install from repository with `--editable` option, in requirements: `-e git+https://github.com/celery/django-celery-beat.git#egg=django_celery_beat`.
2.  In Django `LOCALE_PATHS` setting, I've added the path to django_celery_beat `locale/` directory (located inside `src/` folder of my virtual environment) using:

```python
import os
import django_celery_beat

LOCALE_PATHS = (
    ...,
    os.path.abspath(os.path.join(os.path.dirname(django_celery_beat.__path__[0]), "locale")),
)
```

It's not the best solution but much better than download files, create `locale/` directory, move them inside it... So I will close this for now.
